src/abstractdataframe/iteration.jl

```diff
@@ -190,6 +204,13 @@ Get a vector of column names of `dfc` as `Symbol`s.
 """
 Base.keys(itr::DataFrameColumns) = propertynames(itr)
 
+"""
+    values(dfc::DataFrameColumns)
+
+Get a vector of columns from `dfc`.
+"""
+Base.values(itr::DataFrameColumns) = collect(itr)
```

Comment:
Would this definition be better? It could make sense to use it instead so that the eltype is always the same. People can use `collect` if they prefer the narrowest type.
```suggestion
Base.values(itr::DataFrameColumns) = collect(AbstractVector, itr)
```

Reply:
This is not needed, as `eltype(itr)` is already `AbstractVector`. To make the narrowest type one has to use comprehension `[v for v in itr]`

Reply:
So maybe it would be useful to have `value` always return a `Vector{AbstractVector}`, and `collect` choose the most specific type? Or maybe not. Cf. the discussion above about `IteratorEltype`.